frontend/public/components/storage/shared.ts

```diff
@@ -29,30 +30,33 @@ export const provisionerAccessModeMapping = {
   'pd.csi.storage.gke.io': ['ReadWriteOnce'],
 };
 export const initialAccessModes = ['ReadWriteOnce', 'ReadWriteMany', 'ReadOnlyMany'];
-export const accessModeRadios = [
+
+export const accessModeRadios: any = () => [
```

Comment:
Use a better return type than `any`. If you leave the return type off, TypeScript would infer something better here.

Reply:
Unfortunately it complains about not having access to filter or map. :) I tried a fancier type and it complained about not having access to pop, etc. I can keep trying more types.